packages/cli/command_spec.pony

```diff
@@ -325,22 +343,66 @@ class val ArgSpec
     _name + "[" + _typ.string() + "]" +
       if not _required then "(=" + _default.string() + ")" else "" end
 
-type _Value is (Bool | String | I64 | F64)
+class _StringSeq is ReadSeq[String]
+  let strings: Array[String]
+
+  new val empty() =>
+    strings = Array[String]
+
+  new val from(s: String) =>
+    strings = Array[String].init(s, 1)
+
+  new val combined(ss0: _StringSeq val, ss1: _StringSeq val) =>
+    let ss = ss0.strings.clone()
+    ss.append(ss1.strings)
+    strings = ss
+
+  fun string(): String =>
+    let str = recover trn String() end
+    str.append("[")
+    for s in strings.values() do
+      if str.size() > 1 then str.append(",") end
+      str.append(s)
+    end
+    str.append("]")
```

Comment:
For each of these `append` calls where you're appending a single-character string, you could be `push`ing just the single character (as a `U8`) instead.  For example: `str.push(']')`